Add background shape doc (#299)
dkanada/frost: README.md

Help with any aspect of the app is much appreciated! **You don't have to know how to code!** If you know how to work with vectors in a program that supports the SVG format. [Inkscape](https://inkscape.org/en/) is recommended but not necessary. Please read the [guide on contributing](CONTRIBUTING.md) before making big changes! It also includes some personal preferences on how to work with Inkscape. If you don't use Inkscape you can just ignore them.

When you bring an icon to Frost try to make it minimal.

> The easiest and most noticeable way to do it is to set an icon free of its background.
> Twitter app is a good example!
> There are two parts: a bird and a circle surrounding it.
> The bird is a center of an icon and the part associated with twitter while the circle is not unique to the platform and can be removed.
> Don't take as a ground rule because there are cases when this background shape is necessary.
> Adobe products are much more recognizable in a square while GitHub official uses background shape as a meaningful part of an icon.
> Currently, some of the existing icons don't follow this rule, we are working on it.

Reviewer: ```suggestion
The easiest and most noticeable way to do so is to set an icon free of its background.
Twitter is a good example!
There are two parts: a bird and a circle surrounding it.
The bird is the center of the icon and the part associated with Twitter while the circle is not unique to the platform and can be removed.
Don't take this as a ground rule because there are cases when a background shape is necessary.
Adobe products are much more recognizable in a square and GitHub uses the background shape as a meaningful part of their icon.
Currently, some of the existing icons don't follow this rule, but we are working on improving this.
```
